src/main/scala/santa/ping/Ping.scala

```diff
@@ -19,4 +19,4 @@ object Ping {
   def init(event: FMLInitializationEvent): Unit = {
     MinecraftForge.EVENT_BUS.register(new PingEventHandler())
   }
-}
```

Comment:
Can you undo this change? I'd rather keep this commit only changing the Mod annotation

Reply:
Sure

Reply:
Seems github editor does it automatically and wont let me put it back.

Reply:
Done